test/views/html/paye/historicPayAsYouEarnSpec.scala

```diff
@@ -35,8 +35,7 @@ class historicPayAsYouEarnSpec extends TaiViewSpec with TaxPeriodLabelService{
   private val cyMinusThreeTaxYear: TaxYear = TaxYear(currentYear - 3)
   private val cyMinusFourTaxYear: TaxYear = TaxYear(currentYear - 4)
 
-  private val employment: EmploymentViewModel = EmploymentViewModel("test employment", 123.32, 1, false)
-
+  private val employment: EmploymentViewModel = EmploymentViewModel("test employment", 123.32, 1, false,Some("PAYROLLNO"))
```

Comment:
try and stick to camelCase if possible

Reply:
Not sure which word you are referring to

Reply:
done